llvm/test/Analysis/CostModel/X86/switch.ll

```diff
@@ -0,0 +1,547 @@
+; NOTE: Assertions have been autogenerated by utils/update_analyze_test_checks.py UTC_ARGS: --version 6
```

Comment:
Can you try passing `--filter switch` to UTC? Otherwise the output here is very noisy with all the return costs.